### ISSUE TITLE
Capturing Gradle project does not work for the path with spaces

### ISSUE THREAD

**Issue by nuald** [facebook/infer#669]
**Scope**: either stable or the current development version
```
> infer/bin/infer --version
Infer version v0.12.0-572a383f
```
**STR** (assumes macOS, but it should be reproducible for all platforms):
```
> mkdir "$TMPDIR/a b"
> cp -r examples/android_hello/ "$TMPDIR/a b"
> infer/bin/infer -- "$TMPDIR/a b"/gradlew -b "$TMPDIR/a b/build.gradle" build
...
ERROR:
*** Failed to execute compilation command: exited with code 2
...
*** Output:
javac: file not found: b/app/src/main/java/infer/inferandroidexample/Generated.java
```
**Note**
The PR is under way, it's rather trivial issue to fix.

**Comment by jeremydubreil** [comment]
@nuald: Yes, it is possible that we never encounter this case before. Thanks for trying to send a PR.

**Comment by jvillard** [comment]
Fixed by c2f7ee9.
